src/Sarif/Converters/FxCopConverter.cs

```diff
@@ -131,13 +131,75 @@ internal Result CreateResult(FxCopLogReader.Context context)
 
             result.Locations = new List<Location> { location };
 
-            TryAddProperty(result, context.Level, "Level");
+            bool mapsDirectlyToSarifName;
+
+            result.Level = ConvertFxCopLevelToResultLevel(context.Level, out mapsDirectlyToSarifName);
+
+            if (!mapsDirectlyToSarifName)
+            {
+                // We will not recapitulate FxCop MessageLevel names (such as 
+                // "Error" and "Warning") as a property. For names that differ
+                // (such as "CriticalWarning" and "Information"), we will also 
+                // include the FxCop-specific values in the property bag.
+                TryAddProperty(result, context.Level, "Level");
```

Comment:
Maybe call the property "`FxCopLevel`" to avoid confusion about why there are two properties (one a first-class property, one in the property bag) both named level, and to make it clear that this is "FxCop's notion of 'level'".